Validation/RecoTau/interface/TauValidationMiniAOD.h

```diff
@@ -25,6 +28,9 @@
 #include "DataFormats/Math/interface/LorentzVector.h"
 #include "DataFormats/TauReco/interface/PFTau.h"
 #include "DataFormats/PatCandidates/interface/Tau.h"
+#include "DataFormats/HepMCCandidate/interface/GenParticle.h"
+#include "DataFormats/JetReco/interface/GenJet.h"
+#include "DataFormats/PatCandidates/interface/PackedGenParticle.h"
 #include "DataFormats/Math/interface/deltaR.h"
```

Comment:
Not needed here: please move to the configuration file.
A few other include's can be more conveniently moved there, for example MessageLogger (but perhaps also a few more)

Reply:
Done